pkg/storage/buckets/store.go

```diff
@@ -4,21 +4,25 @@
 package buckets
 
 import (
-	"bytes"
 	"context"
 	"strconv"
 	"time"
 
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/timestamp"
 	"github.com/zeebo/errs"
+	"go.uber.org/zap"
 	monkit "gopkg.in/spacemonkeygo/monkit.v2"
 
 	"storj.io/storj/pkg/encryption"
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/storage/meta"
 	"storj.io/storj/pkg/storage/objects"
+	"storj.io/storj/pkg/storage/segments"
 	"storj.io/storj/pkg/storage/streams"
 	"storj.io/storj/pkg/storj"
-	"storj.io/storj/storage"
+	"storj.io/storj/uplink/metainfo"
```

Comment:
Yeah it is interesting that the metainfo package is nested under the uplink folder, but since it technically is a separate package that doesn't rely on uplink, I think it's ok for now.

Reply:
The `pkg/storage/*` (with minor exceptions) should be under `storj/uplink` as well. So, `uplink/metainfo` is in the right place.